trino-connector/src/test/java/com/datastrato/gravitino/trino/connector/TestGravitinoTableHandle.java

```diff
@@ -10,8 +10,10 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.airlift.json.JsonCodec;
 import io.trino.spi.connector.ConnectorTableHandle;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+@Disabled
```

Comment:
why disable this test? please add a todo comment or open a new issue to track this